src/openrct2/interface/window.h

```diff
@@ -35,6 +35,10 @@ struct TitleSequence;
 struct TextInputSession;
 typedef struct scenario_index_entry scenario_index_entry;
 
+#define ROW_HEIGHT 12
```

Comment:
Perhaps rename this this SCROLLABLE_ROW_HEIGHT?

Reply:
Could do, but then I'll need another one for the non-scrollable ones — like in the list of active marketing campaigns. What do you think?

Reply:
Sounds like a good idea. After all, those are different widgets.

Reply:
Done.